quinn/src/lib.rs

```diff
@@ -155,8 +155,9 @@ enum EndpointEvent {
     Transmit(proto::Transmit),
 }
 
-/// Maximum number of send/recv calls to make before moving on to other processing
+/// Maximum number of datagrams processed in send/recv calls to make before moving on to other processing
 ///
-/// This helps ensure we don't starve anything when the CPU is slower than the link. Value selected
-/// more or less arbitrarily.
-const IO_LOOP_BOUND: usize = 10;
+/// This helps ensure we don't starve anything when the CPU is slower than the link.
+/// Value is selected by picking a low number which didn't degrade throughput in benchmarks.
+/// It is also a multiple of `platform::BATCH_SIZE`
```

Comment:
I'm not sure what the last sentence here means. The actual value is a constant independent of `platform::BATCH_SIZE`, and there doesn't seem to be a strict need for it to be otherwise. Is it meant to express that the *effective* limit may differ from the constant by rounding up to the next batch size?

Reply:
I guess I can follow that its not obvious. The idea in having a multiple is to say "we send exactly N batches", and not N.2 batches. I first had `IO_LOOP_BOUND: usize = BATCH_SIZE * 5`, but that is obviously wrong for `BATCH_SIZE == 1`. 

Then I wondered whether I should redefine `BATCH_SIZE` on top of `IO_LOOP_BOUND`, but that is also wrong. So this just ended up being a comment. I guess we can also remove it

Reply:
Since we break after sending `>=` the bound, I think it's fair to say that the effective size is rounded up to the next multiple of the batch size? Leaving it unstated is fine too, I don't think this is a property that specifically requires documentation.

Reply:
ok. Removed